README.md

```diff
@@ -1,44 +1,59 @@
 # Prisma e2e Tests
 
+This repository continuously tests Prisma Client on and with various operating systems, databases, frameworks, platforms and other setups.
+
 | CI Status                                                                                                                                                                                      | Branch        |
 | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| [![test dev](https://github.com/prisma/e2e-tests/workflows/test/badge.svg?branch=dev)](https://github.com/prisma/e2e-tests/actions?query=workflow%3Atest+-branch%3Apatch-dev+branch%3Adev) <!-- Attention: The `patch-dev` here is intentional, it is _filtering out_ the `patch-dev` branch - otherwise this link will show runs both for `dev` _and_ `patch-dev`. GitHub unfortunately has a (reported) bug in display the filter in the resulting page, so it look as if we were filtering _for_ `patch-dev`. The results are correct though. --> | `dev` |
+| [![test dev](https://github.com/prisma/e2e-tests/workflows/test/badge.svg?branch=dev)](https://github.com/prisma/e2e-tests/actions?query=workflow%3Atest+-branch%3Apatch-dev+branch%3Adev) <!-- Attention: The `patch-dev` in this string is intentional, it is _filtering out_ the `patch-dev` branch - otherwise this link will show runs both for `dev` _and_ `patch-dev`. GitHub unfortunately has a (reported) bug in display the filter in the resulting page, so it look as if we were filtering _for_ `patch-dev`. The results are correct though. --> | `dev` |
```

Comment:
The actual filter looks like it's filtering dev in the end for me
<img width="888" alt="Screen Shot 2021-12-03 at 11 50 52" src="https://user-images.githubusercontent.com/1328733/144590553-e7f1e6ae-e281-4b1b-aa5c-daf8b64bbdce.png">
So maybe "GitHub unfortunately has a (reported) bug in display the filter in the resulting page" is outdated?

Reply:
Indeed, looks like searching for `branch:dev` now only searches for that branch.

Reply:
```suggestion
| [![test dev](https://github.com/prisma/e2e-tests/workflows/test/badge.svg?branch=dev)](https://github.com/prisma/e2e-tests/actions?query=workflow%3Atest+branch%3Adev) | `dev` |
```